docker/pravega/scripts/lts_recovery_backup.sh

```diff
@@ -83,6 +83,7 @@ done
 if [ $container_flushed -eq 1 ]
 then
   echo "Container flushed successfully."
+  sleep 120s
```

Comment:
Rather than simply sleeping, this should really check on progress or the status in some way.
Also why does it echo "Container flushed successfully." before it has been done.